internal/service/plan.go

```diff
@@ -41,13 +42,15 @@ type PlanService interface {
 
 type planService struct {
 	ServiceParams
+	client postgres.IClient
 }
 
 func NewPlanService(
-	serviceParams ServiceParams,
+	params ServiceParams, client postgres.IClient,
```

Comment:
client is now required here
should be plan repo and client is only being used in repo?